inference/inference_basicvsr.py

```diff
@@ -0,0 +1,94 @@
+import argparse
+import cv2
+import glob
+import numpy as np
+import os
+import shutil
+import torch
+
+from basicsr.archs.basicvsr_arch import BasicVSR
+from basicsr.utils.img_util import img2tensor, tensor2img
+
+
+def cache_imgs(paths, device='cpu'):
+    imgs = []
+    imgnames = []
+    for idx, path in enumerate(paths):
+        imgname = os.path.splitext(os.path.basename(path))[0]
+        print('Testing', idx, imgname)
+        img = cv2.imread(path).astype(np.float32) / 255.
+        imgs.append(img)
+        imgnames.append(imgname)
+
+    imgs = img2tensor(imgs, bgr2rgb=True, float32=True)
+    imgs = torch.stack(imgs, dim=0)
+    imgs = imgs.unsqueeze(0).to(device)
+    return imgs, imgnames
```

Comment:
use `read_img_seq` function in `data/data_util.py` 

After replace, you may need to have a test to verify that there is no bug